### DISCUSSION TITLE
Configure mail notification for Code scanning alerts

### DISCUSSION THREAD

**Discussion by thainro** [github/codeql#6663] [Q&A] [answered]
Is there option to configure mail notification for Code scanning alerts. I have disabled the code scanning for each Pull Request & Push. Currently code scanning is done at scheduled time. In that case, if there is any code scanning alert, we won't be able to get those information without checking the security tab. if there is an option to configure mail notification for Code scanning alerts, that will be great.

**Comment by RasmusWL** [comment]
Hi @thainro. I'm not an expert on this area of Code Scanning, but it is my understanding that this is currently not possible. I've reached out to the relevant team, just to double check (but don't get your hopes up).

**Comment by chrisgavin** [comment, marked as answer]
Hi @thainro. There's not currently an option to receive email notifications for new Code Scanning alerts, however it is possible to configure webhooks that can feed other automation. We might add email notifications for new alerts in the future though, as that does indeed seem like a useful feature.

**Reply by tekema-ferc** [reply]
It's been 3 years since this was posted, do we have this feature currently?
